<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет спектра кодовых слов (С использованием CUDA)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="widget.qrc">
    <normaloff>:/iconCUDA.png</normaloff>:/iconCUDA.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="matrixLBL">
     <property name="text">
      <string>Матрица кодовых слов:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FilterPlainTextEdit" name="matrixPTE">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="spectrumLBL">
     <property name="text">
      <string>Спектр кодовых слов:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="spectrumPTE">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCustomPlot" name="spectrumCPT" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="executePBN">
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitPBN">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>filterplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="widget.qrc"/>
 </resources>
 <connections/>
</ui>
